src/pages/Product.jsx

```diff
@@ -0,0 +1,181 @@
+                                <span className="product-details__info__item__list__item__size">
+                                {productInfo?.tags?.join(', ')}
+                                </span>
+                            </div>
+                        }
+                    </div>
+                </div>
+
+          <div className="product-details__info__item">
+                    <div className="product-details__info__item__title">
+                        Quantity
+                    </div>
+                    <div className="product-details__info__item__quantity">
+                        <div className="product-details__info__item__quantity__btn" 
+                            onClick={() => updateQuantity('minus')}>
+                            <i className="bx bx-minus" />
+                        </div>
+                        <div className="product-details__info__item__quantity__input">
+                            {quantity}
+                        </div>
+                        <div className="product-details__info__item__quantity__btn" 
+                            onClick={() => updateQuantity('plus')}>
+                            <i className="bx bx-plus" />
+                        </div>
+                    </div>
+                </div>
+
+                <div className="product-details__info__item">
+                    <Button>Add To Car</Button>
+                </div>
+
+                <div className="product-details__info__item">
+                <div className="product-details__info__item__title">
+                      Specs
+                    </div>
+                    <table className='product-details__info__item__specs'>
+                      <tbody>
+                      {
+                        productInfo?.data?.specs?.map((item, index) => (
+                          <tr key={index}>
+                            <td>{item.spec_name}</td>
+                            <td>{item.spec_value}</td>
+                          </tr>
+                        ))
+                      }
+                      </tbody>
+                    </table>
+                </div>
+
+                <div className={`product-description mobile ${!descriptionExpand ? 'expand' : ''}`}>
+                <div className="product-description__title">
+                    Description
+                </div>
+                <div className="product-description__content" 
+                  dangerouslySetInnerHTML={{__html: productInfo?.data?.description[0].text}} />
+                <div className="product-description__toggle">
+                    <Button size="sm" onClick={() => setDescriptionExpand(!descriptionExpand)}>
+                        {
+                            descriptionExpand ? 'Less' : 'More'
+                        }
+                    </Button>
+                </div>
+            </div>
+      </div>
+		</div>
+}
+
+export default Product;
```

Comment:
![image](https://user-images.githubusercontent.com/26983421/159059533-43b122dc-cb14-4c23-a15f-c1042cef9f33.png)
Take care with responsive views, maybe you can add a little bit of padding